### PR TITLE
Fix: Connect/Onboarding responsiveness

### LINKED ISSUE #1950: 📲 Improve responsiveness
### Feature description

Alby is currently not optimized for usage on smaller screens such as on mobile phones. Browsers with extension support (such as Kiwi Browser) potentially allow users to use Alby on their mobile devices, too.

### Navigation

Discover, Websites, Send & Receive are too wide and break the overall layout. 

image

### Onboarding

Optimize for mobile views:

image

image

### Body height

Seems not to use the full height. Also transaction lists are cut off at the bottom of the bottom of the button.

image

### PR DESCRIPTION
**Pull request by rolznz**
### Describe the changes you have made in this PR

Improve mobile responsiveness, mostly on connect pages (onboarding / login / add accounts)
- moved image/video underneath title for mobile view in connector screens and set password screen
- fix choose connector path page responsive styling
- fix connector grid styling
- fix connector form widths
- fix margins/spacing
- stack buttons vertically and make full width
- some text centering

### Link this PR to an issue [optional]

Fixes #1950

### Type of change

(Remove other not matching type)

- `fix`: Bug fix (non-breaking change which fixes an issue)

### Screenshots of the changes [optional]

Individual connection - image
chrome-extension___lkmjhcdieahekgokeagajaoldpjoofdi_options html(iPhone SE)

Individual connection - video

chrome-extension___lkmjhcdieahekgokeagajaoldpjoofdi_options html(iPhone SE) (1)

Connections grid
chrome-extension___lkmjhcdieahekgokeagajaoldpjoofdi_welcome html(iPhone SE) (7)

Login to your Alby account
chrome-extension___lkmjhcdieahekgokeagajaoldpjoofdi_options html(iPhone SE) (3)

New Alby account
chrome-extension___lkmjhcdieahekgokeagajaoldpjoofdi_options html(iPhone SE) (2)

chrome-extension___lkmjhcdieahekgokeagajaoldpjoofdi_welcome html(iPhone SE) (5)

Choose Connector
chrome-extension___lkmjhcdieahekgokeagajaoldpjoofdi_welcome html(iPhone SE) (2)

chrome-extension___lkmjhcdieahekgokeagajaoldpjoofdi_welcome html(iPhone SE) (3)

Choose Unlock password
chrome-extension___lkmjhcdieahekgokeagajaoldpjoofdi_welcome html(iPhone SE)

chrome-extension___lkmjhcdieahekgokeagajaoldpjoofdi_welcome html(iPhone SE) (1)

_Add screenshots to make your changes easier to understand. You can also add a video here._

### How has this been tested?

_Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration_

### Checklist

- [ ] My code follows the style guidelines of this project and performed a self-review of my own code
- [ ] New and existing tests pass locally with my changes
- [ ] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Comment by rolznz** [comment]
Notes:
- moved image/video underneath title for mobile view in connector screens and set password screen
- fix choose connector path page responsive styling
- fix connector grid styling
- fix connector form widths
- fix margins/spacing
- stack buttons vertically and make full width
- some text centering

**Review by escapedcat — APPROVED** [review]
ack

File: src/app/screens/Onboard/SetPassword/index.tsx [hunk 1]
```diff
@@ -32,29 +32,36 @@ export default function SetPassword() {
...
+          <div className="lg:w-4/5 mb-6">
             <PasswordForm
               i18nKeyPrefix="welcome.set_password"
               formData={formData}
               setFormData={setFormData}
             />
           </div>
         </div>
-        <div className="mt-16 lg:mt-0 lg:w-1/2">
+        <div className="hidden mt-16 lg:mt-0 lg:w-1/2">
```
Comment (bumi) [inline src/app/screens/Onboard/SetPassword/index.tsx:62]: why hidden?

Comment (rolznz) [reply src/app/screens/Onboard/SetPassword/index.tsx]: oops, fixed. Thanks!

**Comment by rolznz** [comment]
Do you guys think this should be the connector image? or a combination of the connector and alby?
image

**Review by reneaaron — APPROVED** [review]
tACK

**Pull request merged**